doc/integrator/create_application.rst.tmpl

```diff
@@ -120,12 +120,14 @@ Simple application
 
 A simple application is an application for which no custom code is needed (Python or JavaScript).
 
+See also :ref:`integration_structure` for more informations.
+
+All the administrator in integrator guides indicate if it doesn't consern the simple application mode.
```

Comment:
I don't understand this sentence

Reply:
Better?:
All the administrator and integrator guides indicate if the section doesn't consern the simple
application mode.